experimental/CliffordAlgOrd/src/CliffordOrder.jl

```diff
@@ -181,14 +181,50 @@ is_exact_type(::Type{ZZCliffordOrderElem}) = true
 @doc raw"""
     clifford_order(ls::QuadLat) -> CliffordOrder
 
-Return the Clifford order of the even quadratic lattice $ls$.
+Return the Clifford order of the even lattice $ls$.
+
+# Examples
+
+```jldoctest
+julia> K,a = quadratic_field(-5); OK = maximal_order(K);
+
+julia> C = clifford_order(lattice(quadratic_space(K, 2*identity_matrix(K, 2))))
+Clifford order of even lattice over maximal order of imaginary quadratic field defined by x^2 + 5
+with basis [1, sqrt(-5)] with Gram matrix
```

Comment:
this seems a bit too long for me here. maybe @thofma knows how to pretty-print these hecke types

Reply:
https://github.com/thofma/Hecke.jl/pull/1840